setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = fortls
+url = https://github.com/gnikit/fortran-language-server
+author = Giannis Nikiteas
+author_email = giannis.nikiteas@gmail.com
+description = fortls - Fortran Language Server
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Fortran
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: MacOS
+project_urls =
+    Source = https://github.com/gnikit/fortran-language-server
+    Tracker = https://github.com/gnikit/fortran-language-server/issues
+
+[options]
+packages = find:
+python_requires = >= 3.7
+install_requires =
+    importlib-metadata; python_version < "3.8"
+
+[options.package_data]
+fortls = *.json
+
+[options.entry_points]
+console_scripts =
+    fortls = fortls.__init__:main
+
+[options.extras_require]
+tests = 
+    pytest >= 5.4.3
+    pytest-cov >= 2.12.1
+    black
+
 [flake8]
-max-line-length = 88
-extend-ignore = E203,E722
\ No newline at end of file
+max-line-length  =  88
+extend-ignore  =  E203, E722
```

Comment:
why the use of double spacing?

Reply:
Drive-by change when moving stuff from setup.py. Will fix!